### ISSUE TITLE
A11y_Visual Studio Code_Download_Keyboard: Keyboard focus is not visible on installer types (64 bit or 34 bit) for all platforms

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4516]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#WCAG2.4.7#Focus Visible#Visual Studio Code Services#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Repro Steps:

1. Hit [Visual Studio Code ](https://code.visualstudio.com/)link
2. Navigate to "Download" button.
3. Tab till installer type links for any platform(Windows/Linux/MacOS).
3. Verify whether keyboard focus is visible on installer type links or not

### Actual Result:
Keyboard focus is not visible on installer types(64 bit or 34 bit) for all platforms

### Expected Result:
Keyboard focus should be clearly visible on installer types(64 bit or 34 bit) for all platforms. Keyboard focus should be clearly visible on all interactive controls in the page

### User Impact:
Keyboard users may not understand where the focus is landing and might get confused

### Attachment:

[A11y_Visual Studio Code_Download_Keyboard.zip](https://github.com/microsoft/vscode-docs/files/6420573/A11y_Visual.Studio.Code_Download_Keyboard.zip)

**Comment by kalpanah1** [comment]
#Closed;
Verified the issue in the below environment. The issue is fixed. Please find the attachment for reference.
URL: https://code.visualstudio.com/Download.
[Bug_4516_Fixed.zip](https://github.com/microsoft/vscode-docs/files/10696886/Bug_4516_Fixed.zip)
